### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Writerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1467,3 +1467,9 @@ We wish you a good studing :3
 ## Hello everyone!
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
+
+***
```
Comment (nazmariam) [inline README.md:1471]: Hi! Good job! Why did you close previous one?

Comment (Writerman) [reply README.md:1471]: Hi! I create new branch with out correcting mispelld

Comment (miroslav-kolomiets) [reply README.md:1471]: For the future, it was not necessary. All such changes must be made in the same branch. You just add another commit to the same PR.

**Review by miroslav-kolomiets — COMMENTED** [review]
For example, now in this branch you need to fix the conflict. You do not need to do a new PR for this. This should be done in the same branch.

**Comment by Writerman** [comment]
Should I do sth else to add my changes to readme?

**Comment by miroslav-kolomiets** [comment]
> Should I do sth else to add my changes to readme?

It's good now. But conflicts need to be resolved. After that we will merge your PR.

**Comment by Writerman** [comment]
Hi!
Now it hasn't conflicts)

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Look at your "File changed", it's here: https://github.com/kottans/mock-repo/pull/465/files
You have there +98 new lines of code, though all that should be there is your greeting. 
Apparently, the conflict was not resolved correctly.

**Pull request closed without merging**